bin/check-event-json

```diff
@@ -1,15 +1,23 @@
 #!/usr/bin/env node
 
-var fs = require("fs");
-var filename = process.argv[2];
+var fs = require('fs');
+var file = process.argv[2];
 
-if (filename) {
+if(file !== undefined && file !== null) {
+  checkJson(file);
+} else {
+  for(var i of ['events.json', '2015/events.json', '2016/events.json']) {
```

Comment:
since they arnt modified in the loop it should prob be const
also suggestion where too put the array or form it im not a big fan of adding for json files.